api/app.py

```diff
@@ -18,23 +15,30 @@
 
 
 @app.route("/")
-def health_check() -> dict:
+def index() -> dict:
     return {
         "data": "Hello, welcome to Sleuth backend interview task. Please see instructions in README.md"
     }
 
 
-@app.route("/health/github")
-def github_api_root_example() -> dict:
-    return GithubREST().get("/")
+@app.route("/health")
+def health_check() -> dict:
+    response = SWApi().get("/")
+    return {"swapi_is_healthy": response.status_code == 200, "urls": response.json()}
 
 
-@app.route("/github/repos/<path:repository>/pulls", methods=["GET"])
-def github_repository_pull_requests(repository: str):
-    # list repos PRs: https://docs.github.com/en/rest/pulls/pulls#list-pull-requests
-    # example of a repository with lots of PRs: https://github.com/django/django
+@app.route("/swapi/<int:min_cargo_capacity>/vehicles", methods=["GET"])
```

Comment:
NOT: this seems very much a query param thing, but if we are not doing that in the name of reducing complexity I would at least swap `min_cargo_capacity` and `vehicles`

Reply:
I'm not sure, I'm following. Do you want to say that it reads better, if it is `"/swapi/vehicles/<int:min_cargo_capacity>"`? 

Yeah, I don't care about the order, I'm fine with whatever looks better.

Reply:
Or that we should get rid of the whole `int:min_cargo_capacity` GET param? And just have 
```py
@app.route("/swapi/vehicles", methods=["GET"])
```